File: LPS_Service/Models/Account/LoanAccountModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace LPS_Service.Models.Account
{
    public class LoanAccountFilterModel
    {
        [Required]
        public int startPage { get; set; }
        [Required]
        public int limitPage { get; set; }
        [Required]
        public string cif { get; set; }
        public string sortBy { get; set; }
        public bool ascending { get; set; }
        public string productGroup { get; set; }
    }
    public class GetLoanAccountModel
    {
        public SpLoanAccountModel[] accounts { get; set; }
        public int totalItems { get; set; }
    }
}

File: LPS_Angular/src/app/models/Account/LoanAccountModel.ts
export class LoanAccountModel {
    ACCTNO: string;
    ACTYPE: string;
    PRODUCT_CODE: string;
    PRODUCT_GROUP: string;
    LIMIT_NO: string;
    OUTSTANDING: number;
    INT_RATE: number;
    ISSUE_DATE: string;
    MATURITY_DATE: string;
    AGING: string;
    LATE_CHARGE: number;
    MISC_CHARGE: number;
    TDR_FLAG: string;
    PROVISION_TFRS9: number;
    PROVISION_DATE: string;
}

export class GetLoanAccountModel {
    accounts: LoanAccountModel[];
    totalItems: number;
}

export class LoanAccountFilterModel {
    startPage: number;
    limitPage: number;
    cif: string;
    sortBy: string;
    ascending: boolean;
    productGroup: string;
}